app/controllers/provider_interface/organisation_permissions_setup_controller.rb

```diff
@@ -125,12 +129,27 @@ def previous_page_path(wizard)
       end
     end
 
-    def send_organisation_permissions_emails(relationships)
+    def send_new_organisation_permissions_emails(relationships)
       relationships.each do |relationship|
         SendOrganisationPermissionsEmails.new(provider_user: current_provider_user, permissions: relationship).call
       end
     end
 
+    def send_setup_organisation_permissions_emails(relationships)
```

Comment:
Could we call this `send_organisation_permissions_updated_emails`, and the other one `send_organisation_permissions_set_up_emails`. It's a bit confusing since we are sending `new` emails for `setup` permissions and `setup` emails for `new` permissions

Reply:
Yep, renamed for clarity